draft-ietf-httpbis-compression-dictionary.md

```diff
@@ -300,12 +300,12 @@ negotiating content encoding in HTTP.
 
 This document introduces two new content encoding algorithms:
 
-|------------------|----------------------------------------------------|
-| Content-Encoding | Description                                        |
-|------------------|----------------------------------------------------|
-| br-d             | Brotli using an external compression dictionary    |
-| zstd-d           | Zstandard using an external compression dictionary |
-|------------------|----------------------------------------------------|
+|------------------|--------------------------------------------------------------------------------------------|
+| Content-Encoding | Description                                                                                |
+|------------------|--------------------------------------------------------------------------------------------|
+| br-d             | Brotli using an external compression dictionary and a compression window of up to 16 MB.   |
+| zstd-d           | Zstandard using an external compression dictionary and a compression window of up to 8 MB. |
+|------------------|--------------------------------------------------------------------------------------------|
```

Comment:
A list might be easier to consume here.  This is also the place where you should include citations.  (though you might also include those citations at the first place that Brotli and Zstandard are mentioned in the intro.